packages/shared/src/components/checklist/ChecklistCard.tsx

```diff
@@ -21,7 +21,7 @@ const ChecklistCard = ({
 
   return (
     <div className={className}>
-      <Card className="p-0 bg-transparent rounded-14 !border-theme-color-cabbage hover:!border-theme-color-cabbage w-[340px]">
+      <Card className=" rounded-14 !p-0 !border-theme-color-cabbage hover:!border-theme-color-cabbage w-[340px]">
```

Comment:
Trailing space.

Reply:
Woops, fixed it :)

Reply:
Whoa just realised Github was slow with posting this since I am sure I sent it before merge 🐌 Tnx for the fix!